src/main/resources/helptexts/languageOfDescription.html

```diff
@@ -1,5 +1,18 @@
-<h1>Test header</h1>
-<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque semper nulla eu placerat consectetur.</p>
-<p>Sed dignissim dapibus venenatis. Sed rhoncus.</p>
-<h2>Subheader2</h2>
-<p>Maecenas luctus metus eleifend, pulvinar mi ac, pellentesque nibh. Sed feugiat, augue.</p>
+<h1>Language</h1>
+<p>The language in which the data and the documents have been written.</p>
```

Comment:
no, this is the language of the Description. I suggest to just only keep the h1 and delete the rest of this helptext here.